src/lib.rs

```diff
@@ -520,4 +520,13 @@ mod test {
             .iter()
             .any(|(_, process)| !process.cmd().is_empty()));
     }
+
+    #[test]
+    fn check_cpu_arch() {
+        if !IS_SUPPORTED {
+            return;
+        }
+        let s = System::new();
+        assert!(s.cpu_arch().is_some());
+    }
```

Comment:
We can implement it in a more interesting way:

```suggestion
        let s = System::new();
        assert_eq!(s.cpu_arch().is_some(), IS_SUPPORTED);
```